### PR TITLE
feat: tag docker with release tag

### PR DESCRIPTION
**Pull request by douglaz**
Let docker-compose files use a tagged release.

### CODE REVIEW

**Comment by elsirion** [comment]
Maybe let's do this after #3043?

**Comment by douglaz** [comment]
Using the new tag

**Review by dpc — APPROVED** [review]

File: docker/full-tls-mutinynet/docker-compose.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ services:
       - "/var/run/docker.sock:/var/run/docker.sock:ro"
 
   fedimintd:
-    image: fedimint/fedimintd:master
+    image: fedimint/fedimintd:v0.1.0-rc3
```
Comment (elsirion) [inline docker/full-tls-mutinynet/docker-compose.yaml:35]: This doesn't look sustainable. I should have thought of that when ACKing #3044. I still think an `v0.1-latest` tag would be better. Sure, as @dpc pointed out the image will get cached, but if you really want the latest version you should probably just manually pull it again.

File: scripts/dev/change-docker-tags.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+if [ -z "$files" ]; then
+  echo "No docker-compose files found in $path"
+  exit 1
+fi
+
+for file in $files; do
+  sed -ri "s/image: fedimint\/(fedimintd|gatewayd):.*/image: fedimint\/\1:$new_tag/g" "$file"
+done
+
```
Comment (elsirion) [inline scripts/dev/change-docker-tags.sh:16]: Waiting for someone to build a workflow that runs this script+opens a PR whenever a tag is created on a release branch :laughing:

**Review by elsirion — APPROVED** [review]
Begrudgingly ACKing till we have a better idea …

File: docker/full-tls-mutinynet/docker-compose.yaml (hunk 1 above)
Comment (douglaz) [reply docker/full-tls-mutinynet/docker-compose.yaml:35]: I think any upgrade to a federation should be done carefully and manually. So here we should specify a very specific tag/hash.

**Pull request merged**